build-system/compile/generate/bento.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * @fileoverview
+ * Compile-time generators of entry-points for Bento-related binaries.
+ */
+
+// TODO(alanorozco): Move generators of extension-related `defineElement()`
+// into this file. Add tests for them, now that we have tests here.
+
+const dedent = require('dedent');
+const {getSharedBentoSymbols} = require('./shared-bento-symbols');
+
+/**
+ * @param {Object<string, string[]>} packageSymbols
+ * @return {string}
+ */
+function generateBentoRuntimeEntrypoint(
+  packageSymbols = getSharedBentoSymbols()
```

Comment:
Optional: Is there a more efficient way to express the default argument than with a function call? Perhaps create a const at the top with the return value and reuse it across this file?